pom.xml

```diff
@@ -22,7 +22,7 @@
     <parent>
         <groupId>org.kill-bill.billing</groupId>
         <artifactId>killbill-oss-parent</artifactId>
-        <version>0.144.66</version>
+        <version>0.144.72-SNAPSHOT</version>
```

Comment:
Let's separate the work to support jdk16 from the migration to byte-buddy. To replace cglib with byte-buddy, I don't think changes in the oss-parent are required? If so, can you revert this change here so that we can ensure CI tests pass first with byte-buddy, while keeping everything else unchanged?

Reply:
It's a good idea to separate the migration to ByteBuddy from JDK 16 support. I did revert this change.

Reply:
I'm looking into issues that are causing a build failure.

Reply:
FYI Looks like it's getting close! Just a few more tests in the jdbi module failing.